### PR TITLE
header footer struture

### PR DESCRIPTION
**Pull request by glenngineer1**
Header/Footer structure for your review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>WAGTASTIC</title>
+  <link rel="stylesheet" type="text/css" href="group1.css">
+
+</head>
+<body>
+
+   <!-- Navigation  -->
+
+  <nav>
```
Comment (aoswalt) [inline index.html]: Nav should be in header tag for organization.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  </nav>
+
+  <!-- Main Image -->
+
+    <!-- <img src="/Images/Winston_1.jpg" alt="Winston" width=200px height=200px> -->
+
+  <!-- Footer -->
+
+  <footer>
+    <h3>Where dogs dreams come true!!!!</h3>
```
Comment (whitney-mitchell) [inline index.html]: dogs' instead of dogs.

**Comment by aoswalt** [comment]
:+1:

**Comment by whitney-mitchell** [comment]
:+1:

**Pull request merged**